src/app/component/CategoryProductList/CategoryProductList.style.scss

```diff
@@ -9,31 +9,30 @@
  * @link https://github.com/scandipwa/base-theme
  */
 
-.CategoryProductList {
-    display: grid;
-    grid-template-columns: repeat(3, calc(33.33% - .66rem));
-    grid-auto-rows: max-content;
-    grid-column-gap: 1rem;
-    padding-bottom: 2rem;
-
-    @include mobile {
-        grid-template-columns: 1fr 1fr;
-        grid-column-gap: 0.5rem;
-        border: 0;
-    }
+:root {
+    --color-load-button-background: var(--color-breadcrumbs-background);
+}
 
+.CategoryProductList {
     @include after-mobile {
-        grid-column: 2;
-        min-height: 200vh;
+        min-height: 200vh;        
     }
-
-    &-Placeholder {
+    
+    &-Page {
         display: grid;
-        grid-template-columns: inherit;
-        grid-column: 1/4;
-
+        grid-template-columns: repeat(3, calc(33.33% - .66rem));
+        grid-auto-rows: max-content;
+        grid-column-gap: 1rem;
+        padding-bottom: 2rem;
+    
         @include mobile {
-            grid-column: span 2;
+            grid-template-columns: 1fr 1fr;
+            grid-column-gap: 0.5rem;
```

Comment:
Use .5rem

Reply:
This one is also related to: https://github.com/scandipwa/base-theme/pull/236#discussion_r308594134